content/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file.md

```diff
@@ -16,18 +16,30 @@ shortTitle: Community health file
 
 ## About default community health files
 
-Default community health files are a set of predefined files that provide guidance and templates for maintaining a healthy and collaborative open source project.
-You can add default community health files to a public repository called `.github`, in the root of the repository or in the `docs` or `.github` folders. These files help you automate and standardize various aspects of your project's development and community interaction. They are essential for promoting transparency, good practices, and collaboration in open source projects.
+Default community health files are a set of predefined files that provide guidance and templates for maintaining a healthy and collaborative open source project. These files help you automate and standardize various aspects of your project's development and community interaction, promoting transparency, good practices, and collaboration.
 
-{% data variables.product.product_name %} will use and display default files for any repository owned by the account that does not have its own file of that type in any of the following places:
+You can add default community health files to a public repository called `.github`. {% data variables.product.product_name %} will use and display default files for any repository owned by the account that does not have its own file of that type. The location of these default files determines whether and how they will be used.
 
-* The root of the repository
-* The `.github` folder
-* The `docs` folder
+### File Location and Precedence
```

Comment:
I'd either skip the heading or move it after "… good practices, collaboration" before starting to talk about any actual files and their locations.